eva/udfs/sift_feature_extractor.py

```diff
@@ -0,0 +1,63 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import List
+
+import cv2
+import kornia
+import numpy as np
+import pandas as pd
+import torch
+
+from eva.udfs.abstract.abstract_udf import AbstractClassifierUDF
+
+
+class SiftFeatureExtractor(AbstractClassifierUDF):
```

Comment:
Please use decorators for the UDF. Moving forward, we want to completely shift to decorators.